sdk/servicebus/service-bus/src/serviceBusMessageBatch.ts

```diff
@@ -123,6 +137,36 @@ export class ServiceBusMessageBatchImpl implements ServiceBusMessageBatch {
     return this._encodedMessages.length;
   }
 
+  /**
+   * Generates an AMQP message that contains the provided encoded messages and annotations.
+   * @param encodedMessages The already encoded messages to include in the AMQP batch.
+   * @param annotations The message annotations to set on the batch.
+   */
+  private _generateBatch(
+    encodedMessages: Buffer[],
+    annotations?: MessageAnnotations,
+    applicationProperties?: Dictionary<any>,
```

Comment:
Is this any different than:
```suggestion
    applicationProperties?: { [key: string]: any },
```
If not, I think we can avoid pulling in the Dictionary type.

Reply:
Done.